src/test/skript/tests/syntaxes/expressions/ExprTagsOfType.sk

```diff
@@ -0,0 +1,7 @@
+test "all tags":
+	assert all tags contains tag "dirt" with "dirt isn't a tag"
+
+	assert block tags contain block tag "slabs" with "oak slab is not a block tag"
+	assert paper item tags contain item tag "paper:doors" with "paper:doors is not an paper item tag"
```

Comment:
```suggestion
	assert paper item tags contain item tag "paper:doors" with "doors is not an paper item tag"
```

Reply:
no this is intentional, it matches the tag being looked for